scripts/toolboxSrc/pushToCDN.ts

```diff
@@ -2,6 +2,11 @@ import fs from 'fs'
 import getFileStoreManager from 'parabol-server/fileStorage/getFileStoreManager'
 import path from 'path'
 import getProjectRoot from '../webpack/utils/getProjectRoot'
+;(require as any).context(
```

Comment:
since the DB needs these files, we require them here so they get added to the server bundle, e.g. `dist/images/templates`